src/main/java/org/example/lesson19/DBConnector/DataBaseConnection.java

```diff
@@ -0,0 +1,25 @@
+package org.example.lesson19.DBConnector;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class DataBaseConnection {
+
+    private Connection connection;
+
+    public void getConnection(String url, String user, String password) throws SQLException {
+        connection = DriverManager.getConnection(url, user, password);
+    }
+
+    public void close() throws SQLException {
```

Comment:
Please, implement AutoClosable interface by this class and override close method to close connection properly